docs/src/app/(public)/(content)/react/utils/direction-provider/page.mdx

```diff
@@ -10,15 +10,19 @@
 
 ## API reference
 
-Import the component and wrap it around your app:
+Import the component and wrap it around your app to enable RTL behavior, and set the `dir` attribute accordingly to affect styling:
 
 ```jsx title="Anatomy"
 import { DirectionProvider } from '@base-ui-components/react/direction-provider';
 
-// prettier-ignore
-<DirectionProvider>
-  {/* Your app or a group of components */}
-</DirectionProvider>
+<html dir="rtl">
+  <body>
+    {/* prettier-ignore */}
+    <DirectionProvider>
+      {/* Your app or a group of components */}
+    </DirectionProvider>
+  </body>
+</html>;
```

Comment:
I'd keep the anatomy and the description as it was before since we don't typically show the API or the required props to make the components work in the Anatomy section. 

I think a code block also might suggest a few things that aren't really true:
- that we expect `dir="rtl"` on `html` specifically
- that Direction Provider depends on `dir`

Also, introducing `<html dir="rtl">` in the anatomy begs to add `direction="rtl"` in the `<DirectionProvider>` below, at which point the docs get kinda muddy 

imo they'll do native `dir` anyway so should be a non-issue